packages/ember-self-focused/addon/templates/components/self-focused.hbs

```diff
@@ -1 +1,4 @@
+<div style="display: none">
```

Comment:
I thought of removing this node after render, but that creates the same problem on the third visit onwards.